### PR TITLE
Resolve type and import issues for TypeScript

### PR DESCRIPTION
**Pull request by justuswilhelm**
SvelteKit v2 uses TypeScript bundler resolution. This required the `types` export to be specified explicitly in the package.json . A similar issue was encountered here: https://github.com/sveltejs/kit/issues/11329

Fortunately, the fix is simple. See commit #48e05c

Having fixed the first issue brought up a second one: `mask-type` is not a valid <mask> attribute. I have fixed this by making it a Svelte style attribute instead. See #c8b931e.

Cheers!

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -1,5 +1,6 @@
 MIT License
 
+Copyright (c) 2023-2024 Justus Perlwitz
```
Comment (dusty-phillips) [inline LICENSE:3]: Is this intended? I wouldn't normally expect an open source contribution to take over copyright.

File: .editorconfig [hunk 2]
```diff
@@ -0,0 +1,14 @@
+root = true
```
Comment (dusty-phillips) [inline .editorconfig:1]: Not saying that it's wrong to have this file in the repo, but it probably should be in a separate PR.

File: package.json [hunk 3]
```diff
@@ -3,13 +3,21 @@
...
   "types": "dist/ts/index.d.ts",
-  "author": "Paolo Tiu <paolotiu17@gmail.com>",
+  "contributors": [
+    {"name": "Paolo Tiu",
+      "email": "paolotiu17@gmail.com"
+    },
+    {"name": "Justus Perlwitz",
+      "email": "hello@justus.pw"
+    }
+  ],
```
Comment (dusty-phillips) [inline package.json:20]: Perhaps I'm not aware of some offline discussion on repo ownership, but curious if this is acceptable to the original maintainer. If not, are you hosting a fork anywhere?

**Review by dusty-phillips — COMMENTED** [review]
I'd like to see the fix for the types issue released, so I have added some comments about things that may be distracting the maintainer from the primary goal of this PR. That said, I acknowledge that I don't know if you've had any offline discussion about package ownership.

@paolotiu I have tested this branch in my package repo and can confirm that it fixes the sveltkit v2 type error. These are the commands I needed to test it:

```
pnpm add github:jwpconsulting/svelte-boring-avatars#justus/2024-02-19-resolve-import-issue
cd node_modules/svelte-boring-avatars/
npm run build
```

File: package.json (hunk 3 above)
Comment (paolotiu) [reply package.json:20]: Thanks for this @dusty-phillips!! I'll resolve some of these issues then merge the pr

**Comment by paolotiu** [comment]
Merged in [v1.2.6](https://github.com/paolotiu/svelte-boring-avatars/releases/tag/v1.2.6)

**Comment by paolotiu** [comment]
For transparency, these commits were manually removed by me due to the request of the author. I intend no hostility and wish them the best.
